src/webapi/event.rs

```diff
@@ -1221,6 +1223,290 @@ impl ConcreteEvent for PopStateEvent {
     const EVENT_TYPE: &'static str = "popstate";
 }
 
+/// Wrapper type around a CloseEvent code, indicating why the WebSocket was closed
+///
+/// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/CloseEvent)
```

Comment:
Please add a link to the specs (not it the docs; as a normal comment)

https://html.spec.whatwg.org/multipage/web-sockets.html#the-closeevent-interface

Reply:
You mean for the status codes? If so I'll add a link to here: https://tools.ietf.org/html/rfc6455#section-7.4

Reply:
No, I meant the link which I've pasted. (:

In general I'd like to have a direct link to the specs everywhere we can put one. (So, for example, the `CloseEvent` has its own section so we should put a link to it.)